client/boptest.py

```diff
@@ -0,0 +1,262 @@
+import uuid
+import requests
+import json
+import os
+import boto3
+from minio import Minio
+from requests_toolbelt.multipart.encoder import MultipartEncoder
+from shutil import copyfile
+#import getpass
+from pymongo import MongoClient
+import testcase
+
+
+class Boptest:
+
+    # The url argument is the address of the Boptest server
+    # default should be http://localhost/api
+    def __init__(self, url):
+        if url != "http://localhost/api":
```

Comment:
We should allow other URLs. BOPTEST could be deployed on locations other than localhost.